File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Querying/IdListQueryResultTestCase.cs
namespace Db4objects.Db4o.Tests.Common.Querying
{
	public class IdListQueryResultTestCase : Db4objects.Db4o.Tests.Common.Querying.QueryResultTestCase
	{
		public static void Main(string[] args)
		{
			new Db4objects.Db4o.Tests.Common.Querying.IdListQueryResultTestCase().RunSolo();
		}

		protected override Db4objects.Db4o.Inside.Query.AbstractQueryResult NewQueryResult
			()
		{
			return new Db4objects.Db4o.Inside.Query.IdListQueryResult(Trans());
		}
	}
}

File: db4oj.tests/src/com/db4o/db4ounit/common/querying/IdListQueryResultTestCase.java
/* Copyright (C) 2006  db4objects Inc.  http://www.db4o.com */

package com.db4o.db4ounit.common.querying;

import com.db4o.inside.query.*;
import com.db4o.inside.query.result.*;


public class IdListQueryResultTestCase extends QueryResultTestCase {

	public static void main(String[] args) {
		new IdListQueryResultTestCase().runSolo();
	}

	protected AbstractQueryResult newQueryResult() {
		return new IdListQueryResult(trans());
	}

}
